code/game/gamemodes/meteor/meteors.dm

```diff
@@ -344,6 +344,16 @@ GLOBAL_LIST_INIT(meteorsC, list(/obj/effect/meteor/dust)) //for space dust event
 	..()
 	if(prob(20))
 		explosion(src.loc,2,4,6,8)
+		
+/obj/effect/meteor/guardian
+	name = "glowing meteor"
+	icon_state = "glowing"
+	hits = 3
+	heavy = 1
+	meteorsound = 'sound/effects/bamf.ogg'
+	meteordrop = list(/obj/item/stand_arrow)
+	dropamt = 1
+	threat = 100
```

Comment:
modularise

Reply:
oh shit i forgot about that

Reply:
but seriously all this does is give the server a chance to spawn stand arrows through incredibly dangerous meteors, it doesn't add sprites, it doesn't change any descriptions, i don't think this needs modularising

Reply:
TAKE this code and make a new folder in the AUSTATION folder so the path looks like this
austation/austation/code/game/gamemodes/meteor/meteors.dm and put this code into it

that way, your object file can still be read by the code and it won't make a merge conflict if anyone from bee changes this file.

THEN go into austation includes and add your new austation filepath to includes.dm